lib/reporting/reports/suppliers/base.rb

```diff
@@ -56,17 +54,19 @@ def rules
             {
               group_by: :producer,
               header: true,
-              summary_row: proc do |_key, items|
-                line_items = items.flatten
+              summary_row: proc do |_key, line_items|
+                summary_hash = Hash.new(0)
 
-                {
-                  total_excl_vat_and_fees: total_excl_vat_and_fees.call(line_items),
-                  total_excl_vat: total_excl_vat.call(line_items),
-                  total_fees_excl_vat: total_fees_excl_vat.call(line_items),
-                  total_vat_on_fees: total_vat_on_fees.call(line_items),
-                  total_tax: total_tax.call(line_items),
-                  total: total.call(line_items),
-                }
+                line_items.each do |line_item|
+                  summary_hash[:total_excl_vat_and_fees] += total_excl_vat_and_fees.call(line_item)
+                  summary_hash[:total_excl_vat] += total_excl_vat.call(line_item)
+                  summary_hash[:total_fees_excl_vat] += total_fees_excl_vat.call(line_item)
+                  summary_hash[:total_vat_on_fees] += total_vat_on_fees.call(line_item)
+                  summary_hash[:total_tax] += total_tax.call(line_item)
+                  summary_hash[:total] += total.call(line_item)
+                end
```

Comment:
We are only testing with one order and one line item. I think we need to update the spec to use more than on order and more than one line item in order for this to be properly tested.

Reply:
It has been addressed here: https://github.com/openfoodfoundation/openfoodnetwork/pull/12879/commits/03c8e2cee3ff6dee58185b6f87e498b4c7397c49#diff-36e7c39c383994d08652e0192d25c4f629d776cc106cb7cec3268bbb8cbd59bf